frontend/.eslintrc.js

```diff
@@ -57,7 +57,10 @@ module.exports = {
       rules: {
         camelcase: "off",
         // Prevent dead code accumulation
-        "@typescript-eslint/no-unused-vars": "error",
+        "@typescript-eslint/no-unused-vars": [
+          "error",
+          { argsIgnorePattern: "^_" },
```

Comment:
I've always found this to be a helpful pattern for marking unused variables that are expected, but we can discuss as a team

Reply:
I have no issue with this